evmos/evmos
# chore(release): v10.0-rc2

- fix: Minor adjustments/fixes to the init script (#1051)
- docs(upgrades): update cosmovisor command (#1053)
- ante: remove min validator commision decorator (#1057)
- build(deps): bump go.opencensus.io from 0.23.0 to 0.24.0 (#1048)
- imp(erc20): util functions
- Revert "imp(erc20): util functions"
- docs: add Pokt endpoint (#1052)
- imp(erc20): util functions (#1059)
- build(deps): bump github.com/onsi/ginkgo/v2 from 2.4.0 to 2.5.0 (#1061)
- docs: sort public endpoints from older to latest
- docs: Update telemetry.md with correct url to cosmos telemetry docs (#1063)
- docs: Update evmos binary versions and testnet upgrades + fix dead links + update out-dated links (#1071)
- build(deps): bump github.com/cosmos/ibc-go/v5 from 5.0.1 to 5.1.0 (#1068)
- build(deps): bump github.com/tendermint/tendermint from 0.34.22 to 0.34.23 (#1067)
- feat: amino encoding support for the vesting module (#1070)
- build(deps): bump github.com/onsi/gomega from 1.24.0 to 1.24.1 (#1069)
- build(deps): bump golangci/golangci-lint-action from 3.3.0 to 3.3.1 (#1079)
- refactor(all): refactor errors import to use `cosmossdk.io` pkg (#1080)
- fix(proto): fix package version (#1083)
- feat(ibc): utils for erc20 middleware (#1081)
- feat(ibc): erc20 conversion trough  `transfer` module  (#1085)
- fix: use of home directory in local node script (#1088)
- docs: Add audits page (#1090)
- feature(vesting): add functionality to update vesting funder in ClawbackVestingAccount (#1087)
- docs: add info about use of the home flag for the local_node.sh script to docs (#1091)
- chore(deps) Dependency Bumps [chain-chores] (#1092)
- chore: add protolint yaml file (#1093)
- feat(erc20): OnRecv automatic conversion (#1086)
- fix: unbound metrics and remove labels that keep increasing (#1094)
- fix: protogen scripts were not generating the Any types correctly (#1084)
- build(deps): bump github.com/cosmos/cosmos-sdk from 0.46.5 to 0.46.6 (#1098)
- build(deps): bump github.com/onsi/ginkgo/v2 from 2.5.0 to 2.5.1 (#1096)
- build(deps): bump google.golang.org/grpc from 1.50.1 to 1.51.0 (#1097)
- build(deps): bump github.com/tendermint/tendermint from 0.34.23 to 0.34.24 (#1102)
- fix: Fix tabs in documentation (#1099)
- feat(tests): e2e tests for chain upgrades (#1028)
- chore: add gitleaks ignore (#1108)
- imp(erc20): conversion changes (#1104)
- remove unnesary package.json (#1111)
- chore(app): store listener (#1114)
- build(deps): bump cosmossdk.io/math from 1.0.0-beta.3 to 1.0.0-beta.4 (#1115)
- fix(app): register node service (#1118)
- fix(cmd): add missing GetAuxToFeeCommand (#1117)
- fix(ci): add local_node.sh to gitleaks config (#1110)
- fix(cmd): set snapshot config to application (#1120)
- bug(MAKEFILE): fix `evmosd version` cmd to show either tag or last commit (#1121)
- docs(guides) add geth JS console guide (#1105)
- imp: add option to overwrite in local node script (#1123)
- chore(claims): Delete deprecated migration logic (#1126)
- imp: remove create_empty_blocks and create_empty_blocks_interval (#1132)
- fix: update Keplr docs to remove (beta) (#1133)
- fix: enable HTTP server on localnode by default (#1136)
- chore(docs): add priority mempool docs (#1128)
- feat(ci): Golang dependency vulnerability check (#1138)
- chore: remove simulations (#1139)
- chore(deps): bump Ethermint version to v0.20-rc3 (#1140)
- fix(ante): readd EIP712 decorator (#1141)
- feat(erc20): additional validation and checks from audit (#1119)
- docs: Add Ledger CLI docs (#1135)
- docs(guides): add automated coin conversion guide (#1127)
- style: fix vesting example file (#1143)
- fix: set encoding config from root for EIP-712 verification from key (#1145)
- imp(erc20): ignore gas costs on IBC transfer for automatic ERC20 conversion (#1144)
- Move encoding config setup from command line to app (#1146)
- fix(erc20): automatic conversion using 'erc20/<contract_address>' (#1147)
- feat(eip712): add encoding config init back to root (#1148)
- build(deps): bump golang from 1.19.3-bullseye to 1.19.4-bullseye (#1151)
- build(deps): bump technote-space/get-diff-action from 6.1.1 to 6.1.2 (#1150)
- fix(erc20): update token denom on msg (#1149)
- feat: Implement Ledger support on CLI (#816)
- chore(deps): bump Ethermint to v0.20-rc4 (#1157)



## Description



Closes: #XXX

______

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.

PR review checkboxes:

I have...

- [ ] targeted the correct branch (see [PR Targeting](https://github.com/evmos/evmos/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link in the PR description to the relevant issue or specification
- [ ] confirmed all required CI checks have passed

Code maintenance:

I have...

- [ ] written unit and integration [tests](https://github.com/evmos/evmos/blob/main/CONTRIBUTING.md#testing)
- [ ] added relevant [`godoc`](https://go.dev/blog/godoc) and [code comments](https://blog.jbowen.dev/2019/09/the-magic-of-go-comments/).

______

### Reviewers Checklist

**All** items are required. Please add a note if the item is not applicable and please add your handle next to the items reviewed if you only reviewed selected items.

I have...
